addons/mass_mailing/views/mailing_templates_portal_management.xml

```diff
@@ -23,6 +23,29 @@
             </div>
         </t>
     </template>
+    <template id="mailing_generic_view" name="View in Browser">
```

Comment:
:socks: missing void line, and would use "mailing_view" namespace like the other template, and probably add a quick comment, like
```suggestion

    <!-- Template used for '/view' generic dummy placeholder -->
    <template id="mailing_view_generic" name="Dummy Browser View">
```